.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,ts}]
+charset = utf-8
+
+# Indentation override for all JS under lib directory
+[src/**.ts,index.ts]
+indent_style = space
+indent_size = 2
+
+# Matches the exact files either package.json or .travis.yml
+[{package.json,.travis.yml}]
+indent_style = space
+indent_size = 2
```

Comment:
All these settings will be applicable to most of the files so its worth to move them into `[*]` section.
I would also suggest adding `trim_trailing_whitespace = true`

Reply:
makes sense 👍 
at the beginning worth to have configs as simple as possible